notebooks/data-sources/TestGrid/testgrid_EDA.ipynb

```diff
@@ -13,14 +13,28 @@
     "\n",
```

Comment:
<p>Would it make sense to introduce the word "job" here, since it's being used here for the first time? So It's meaning in the CI world.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>sounds good! please add it : ) </p>

Reply:
It seems like a job is a collection of tests run together for a build? I'm not exactly sure on this...

Reply:
<p>Would it be clearer if we replaced the word "job" with "test suite" here? Since what we are really referring to is a collection of specific tests.  </p>

Reply:
It would be but I guess if we change it now it would make the notebook a little less readable since there are a lot of instances of the word "job" including markdown cells, and variable names in the code. Specifying that job is a collection of tests would be the least messy option at this point, I think.